README.md

```diff
@@ -7,7 +7,7 @@ registered on the local homeserver by querying a remote backend.
 
 From the virtual environment that you use for Synapse, install this module with:
 ```shell
-pip install path/to/synapse-3pid-checker
+pip install threepid-checker==1.0.1
```

Comment:
Is there a reason why you would specifically install `1.0.1` rather than just use the latest version? For now both are the same but it just feels a bit weird to me to specify a version in the `pip install` command here.

Reply:
No reason, I've changed it to be more general